core/domain/story_domain.py

```diff
@@ -152,7 +159,7 @@ class StoryNode(object):
     def __init__(
             self, node_id, destination_node_ids,
             acquired_skill_ids, prerequisite_skill_ids,
-            outline, exploration_id):
+            outline, outlines_are_finalized, exploration_id):
```

Comment:
outline_is_finalized, I think, since the other thing here is called `outline`

Reply:
Done.